charts/test-broker/README.md

```diff
@@ -0,0 +1,49 @@
+# Test Service Broker
+
+Test Service Broker is an example
+[Open Service Broker](https://www.openservicebrokerapi.org/)
+for manualyl testing & demonstrating the Kubernetes
+Service Catalog.
+
+For more information,
+[visit the Service Catalog project on github](https://github.com/kubernetes-incubator/service-catalog).
+
+## Installing the Chart
+
+To install the chart with the release name `test-broker`:
+
+```bash
+$ helm install charts/test-broker --name test-broker --namespace test-broker
+```
+
```

Comment:
probably worth adding that you must register the broker using the associated yaml or `svcat register test-broker --url http://test-broker-test-broker.test-broker.svc.cluster.local`  OR make the helm chart register the broker.

Reply:
I've added it to the Readme. I didn't want to add it to the Helm chart, as it would have to be configurable whether to register a cluster- or a namespace-scoped broker.